package.json

```diff
@@ -59,7 +59,7 @@
     "test:sim:clean:connext": "(cd test/simulation && ./docker-clean.sh connext)",
     "test:sim:clean:custom-xud": "(cd test/simulation && ./docker-clean.sh custom-xud)",
     "test:sim:logs": "cd test/simulation && ./logs.sh",
-    "test:jest": "jest --unhandled-rejections=strict --rootDir test/jest",
+    "test:jest": "jest --forceExit --unhandled-rejections=strict --rootDir test/jest",
```

Comment:
Mainly curious but why `--forceExit` here? I've spent some time fiddling with these jest options but didn't consider this one.

Reply:
`--forceExit` terminates Jest even if there are dangling unresolved Promises. Is there some bug in the changes tests that requires a force exit of Jest?

Reply:
I was experiencing tests passing locally, but running forever in the CI. Didn't want to spend more time debugging this as this seemed to solve the issue.

Reply:
I'll reference this in the running "Tests Flakiness" issue.